test/website-nanostores/pnpm-lock.yaml

```diff
@@ -0,0 +1,3348 @@
+lockfileVersion: '6.0'
```

Comment:
Should we use pnpm monorepo for that instead of having multiple lock files? It could be easier to maintain the single lock file.